cdap-ui/app/directives/log-viewer/log-viewer.less

```diff
@@ -204,26 +207,35 @@ my-log-viewer {
         margin: 0;
 
         .log-action-btn {
-          height: 25px;
-          width: 100px;
-          border: solid 1px white;
+          height: 28px;
+          border: solid 1px #acacac;
           color: white;
           background-color: transparent;
           border-radius: 4px;
           margin-right: 3px;
           outline: grey;
         }
+
+        .download-all {
+          margin-left: 3px;
+          width: 100px;
```

Comment:
We should probably not have width for buttons. We can have max-width and have ellipsis on long labels but in this case we have static labels and its  not too long. =